scene/2d/position_2d.h

```diff
@@ -38,14 +38,20 @@ class Position2D : public Node2D {
 	GDCLASS(Position2D, Node2D)
 
 	float gizmo_length;
+	
```

Comment:
Trailing space.

Reply:
Fixed. Looks like I forgot to clang-format the header.